### PR TITLE
Update Azure Network Watcher app doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/microsoft-azure/network-watcher.md [hunk 1]
```diff
@@ -115,106 +113,97 @@ To configure an HTTP source for Azure, do the following:
...
-* Filter Prefix (Optional): If you want to filter logs from a specific container, enter the following, replacing the variable with your container name:
-```bash
-/blobServices/default/containers/<container_name>/
-```
-
-
-Resource group names should not consist of an underscore.
-
-1. Select the check box to **agree to the terms and conditions**, and then click **Purchase**.<br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure_Blob_Storage_Custom_Deployment.png')} alt="Azure_Blob_Storage_Custom_Deployment" />
+        * SumoEndpointURL: URL for the HTTP source you configured in [Step 2](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage#Step_2._Configure_an_HTTP_source) above.
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/network-watcher.md:130]:
```suggestion
        * SumoEndpointURL: URL for the HTTP source you configured in [Step 2](#step-2-configure-an-http-source) above.
```

File: docs/integrations/microsoft-azure/network-watcher.md [hunk 2]
```diff
@@ -115,106 +113,97 @@ To configure an HTTP source for Azure, do the following:
...
+        * StorageAccountName: Name of the storage account where  you are storing logs from Azure Service, that you configured in [Step 1](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage#Step_1._Configure_Azure_storage_account) above.
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/network-watcher.md:131]:
```suggestion
        * StorageAccountName: Name of the storage account where  you are storing logs from Azure Service, that you configured in [Step 1](#step-1-configure-azure-storage-account) above.
```

File: docs/integrations/microsoft-azure/network-watcher.md [hunk 3]
```diff
@@ -115,106 +113,97 @@ To configure an HTTP source for Azure, do the following:
...
+        * StorageAccountResourceGroupName: Name of the resource group of the storage account you configured in [Step 1](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage#Step_1._Configure_Azure_storage_account) above.
+        * StorageAccountRegion: Name of the region of the storage account you configured in [Step 1](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage#Step_1._Configure_Azure_storage_account) above.
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/network-watcher.md:133]:
```suggestion
        * StorageAccountRegion: Name of the region of the storage account you configured in [Step 1](#step-1-configure-azure-storage-account) above.
```

File: docs/integrations/microsoft-azure/network-watcher.md [hunk 4]
```diff
@@ -115,106 +113,97 @@ To configure an HTTP source for Azure, do the following:
...
 * Specify Timestamp locator as `\"time\": (.*),`
 
-
 ### Step 3: Configure Azure Resources using ARM template
 
-In this step, you use a Sumo-provided Azure Resource Manager (ARM) template to create an Event Hub, three Azure functions, Service Bus Queue, and a Storage Account.
+In this step, you use a Sumo Logic provided Azure Resource Manager (ARM) template to create an Event Hub, three Azure functions, Service Bus Queue, and a Storage Account.
 
 1. Download the [blobreaderdeploy.json](https://raw.githubusercontent.com/SumoLogic/sumologic-azure-function/master/BlockBlobReader/src/blobreaderdeploy.json) ARM template.
 The above template uses Consumption Plan which does not support VNet integration, you can use [blobreaderdeploywithPremiumPlan.json](https://raw.githubusercontent.com/SumoLogic/sumologic-azure-function/master/BlockBlobReader/src/blobreaderdeploywithPremiumPlan.json) which uses Elastic Premium plan.
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/integrations/microsoft-azure/network-watcher.md:121]: @JV0812, I suggested updating the sentence because it didn't make sense.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@JV0812, I found just a few things that need updating.

**Review by jpipkin1 — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

**Pull request merged**